frontend/lib/api/form/pollingstation/pollingStationUtils.ts

```diff
@@ -393,6 +393,13 @@ export function getClientState(formState: FormState, acceptWarnings: boolean, co
   return clientState;
 }
 
+export function getDataEntryProgress(formState: FormState) {
+  const sections = Object.keys(formState.sections);
+  const totalSections = sections.length;
+  const indexFurthest = sections.indexOf(formState.furthest) + 1;
```

Comment:
Not an issue worth fixing I think, but worth noting:

We update `furthest` with the current page, when we save the page. So "Volgende" has a different effect on the progress than "Invoer afbreken" > "Invoer opslaan".

For example: if you start data entry, progress is 0%. If you then save and resume data entry, progress is 13%, because `furthest` was updated by the save.

Reply:
@jorisleker tagging you, so you're aware :point_up_2:

Reply:
Tx. Not worth fixing indeed. But does make me wonder: why doesn't navigating to a form section/page set `furthest` to that section?